### PR TITLE
All Around Bug Fix

### PR DESCRIPTION
**Pull request by hailed748**
Resolves the following bugs:

1. NavBar on every page
2. Logo now links to main page
3. Logo is not centered 
4. Removes "." from "Retrieve Prior Rating" button
5. Titles on all pages (want to make it so title is course name/prof name don't know if I can get this done with time left)
6. Removes "@username" because we decided username is not needed
7. Worked with Laura to fix missing "(" in data servlet test

Would have broken up into smaller PR's but a lot of these were very small or repetitive like adding titles and nav bar

### CODE REVIEW

File: src/main/webapp/course.html [hunk 1]
```diff
@@ -222,13 +228,13 @@ <h1 class="p-4 text-center">Add Your Review</h1>
             <div class="col">
               <p class="lead">Want to Translate to English?</p>
               <select class="custom-select mb-5" id="translate">
-                <option value="false" disabled selected>Wants to translate?</option>
+                <option value="false" disabled selected>
+                  Wants to translate?
```
Comment (lqueipom) [inline src/main/webapp/course.html:232]: Here it is "want" to translate, it is now fixed in master but just wanted to let u know

File: src/main/webapp/login-script.js [hunk 2]
```diff
@@ -8,16 +63,16 @@ async function signIn(googleUser) {
...
-    const selectElement = document.getElementById("search-term");
-    const optionElement = document.createElement("option");
-    optionElement.appendChild(document.createTextNode("Select term..."));
-    optionElement.value = "";
-    selectElement.appendChild(optionElement);
+    const select = document.getElementById("search-term");
+    const opt = document.createElement("option");
+    opt.appendChild(document.createTextNode("Select term..."));
+    opt.value = "";
+    select.appendChild(opt);
```
Comment (ninaprabhu) [inline src/main/webapp/login-script.js:71]: This is undoing the changes I made to this file - is there a way you can uncommit these changes or pull from master before merging this in?

Comment (ninaprabhu) [reply src/main/webapp/login-script.js:71]: This goes for basically this whole file

Comment (hailed748) [reply src/main/webapp/login-script.js:71]: I can pull from master

**Comment by LukeGMTravis** [comment]
Starting review

File: src/main/webapp/login-script.js [hunk 3]
```diff
@@ -1,3 +1,58 @@
+function changeElementSignIn(privateBool) {
```
Comment (LukeGMTravis) [inline src/main/webapp/login-script.js:1]: "changeElement" isn't very descriptive

File: src/main/webapp/search-style.css [hunk 4]
```diff
@@ -82,34 +82,27 @@ body {
...
 .body {
-  align-items: center;
-  background-color: #1a1a1a;
   display: -ms-flexbox;
   display: flex;
   -ms-flex-align: center;
+  align-items: center;
   padding-top: 40px;
   padding-bottom: 40px;
+  background-color: #1a1a1a;
```
Comment (LukeGMTravis) [inline src/main/webapp/search-style.css:92]: Nit: alphabetize

Comment (LukeGMTravis) [reply src/main/webapp/search-style.css:92]: Same elsewhere

**Pull request merged**